MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 include LICENSE
 include README.md
 
+recursive-include data *
```

Comment:
Should the data be in the package?

Reply:
I hava changed it to be excluded for now.